bmds_server/main/settings/base.py

```diff
@@ -79,8 +79,8 @@
     "default": {
         "ENGINE": "django.db.backends.postgresql",
         "NAME": os.getenv("DJANGO_DB_NAME", "bmds-online"),
-        "USER": os.getenv("DJANGO_DB_USER", "bmds-online"),
```

Comment:
remove these lines :). Just create a `bmds_server/main/settings/local.py` and put your custom settings for the database there.

Reply:
removed